### PR TITLE
Additional script tag options added

### PR DESCRIPTION
**Pull request by wiseguy1394**
I've added some more options for the script tag:
https://support.cookiebot.com/hc/en-us/articles/4406571299346-The-Cookiebot-CMP-widget#h_01FGB7S0AF7M41HJC8ZD83MA0F

### CODE REVIEW

**Comment by erichjsonfosse** [comment]
Hi @wiseguy1394 !
Thanks for the PR!

It seems like this PR contains a lot of changes way beyond the scope of "Additional script tag options added". Could you scope it down a bit? :)

**Comment by wiseguy1394** [comment]
@erichjsonfosse ah..sry. I resetted changes which are not for the PR.

File: projects/ngx-cookiebot/src/lib/ngx-cookiebot.service.ts [hunk 1]
```diff
@@ -126,6 +126,22 @@ export class NgxCookiebotService {
...
+    if (this.cookiebotConfig.widgetPosition) {
+      script.setAttribute('data-widget-position', this.cookiebotConfig.widgetPosition);
+    }
+    
+    if (this.cookiebotConfig.widgetDistanceVertical) {
+      script.setAttribute('data-widget-distance-vertical', this.cookiebotConfig.widgetDistanceVertical + '');
+    }
+    
+    if (this.cookiebotConfig.widgetDistanceHorizontal) {
+      script.setAttribute('ddata-widget-distance-horizontal', this.cookiebotConfig.widgetDistanceHorizontal + '');
```
Comment (erichjsonfosse) [inline projects/ngx-cookiebot/src/lib/ngx-cookiebot.service.ts:143]: Typo in attribute name.

File: projects/ngx-cookiebot/src/lib/ngx-cookiebot.service.ts [hunk 2]
```diff
@@ -126,6 +126,22 @@ export class NgxCookiebotService {
...
+    if (this.cookiebotConfig.widgetEnabled) {
+      script.setAttribute('data-widget-enabled', this.cookiebotConfig.widgetEnabled ? 'true' : 'false');
+    }
+    
+    if (this.cookiebotConfig.widgetPosition) {
+      script.setAttribute('data-widget-position', this.cookiebotConfig.widgetPosition);
+    }
+    
+    if (this.cookiebotConfig.widgetDistanceVertical) {
+      script.setAttribute('data-widget-distance-vertical', this.cookiebotConfig.widgetDistanceVertical + '');
```
Comment (erichjsonfosse) [inline projects/ngx-cookiebot/src/lib/ngx-cookiebot.service.ts:139]: Would prefer toString() instead of adding an empty string.

**Review by erichjsonfosse — CHANGES_REQUESTED** [review]
Thanks for contributing!
See comments :+1:

**Review by erichjsonfosse — APPROVED** [review]

**Comment by erichjsonfosse** [comment]
@wiseguy1394 
Nice!
Thank you :)
We'll merge and publish a new version asap.

**Comment by erichjsonfosse** [comment]
@wiseguy1394 
I've published an rc with the latest changes now (https://www.npmjs.com/package/@halloverden/ngx-cookiebot/v/2.1.0-rc.1). Could you test it and see if it is as you expect :)?

**Comment by wiseguy1394** [comment]
It works as expected. Thx. 👍🏻

**Comment by erichjsonfosse** [comment]
Great!
New version published https://www.npmjs.com/package/@halloverden/ngx-cookiebot/v/2.2.0

**Pull request merged**